NOTES.md

```diff
@@ -0,0 +1,12 @@
+# Problems with Pypy
```

Comment:
it really doesn't help to create files like this.

If you have notes to make, please put them in on the PR.